core/update.php

```diff
@@ -342,6 +316,117 @@ function update_info_page() {
   return $output;
 }
 
+/**
+ * Provides a form to create an on-demand backup before updating.
+ *
+ * @return string
+ *   Rendered HTML form.
+ */
+function update_backup_page() {
+  update_task_list('backup');
+  backdrop_set_title('Pre-update backup');
+
+  $elements = backdrop_get_form('update_backup_form');
+  return backdrop_render($elements);
+}
+
+
+/**
+ * Form constructor for the list of available database module updates.
+ */
+function update_backup_form($form, &$form_state) {
+  // Check if Backup directory is specified. If FALSE, this form shouldn't even
+  // be displayed. If NULL, link to documentation on setting up backups.
```

Comment:
Still pending: actually skip this whole form is the setting's FALSE. :wink:

Reply:
This is now in place.